recipes/yasm/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, MSBuild
+import os
+
+
+class YASMInstallerConan(ConanFile):
+    name = "yasm"
+    version = "1.3.0"
+    url = "https://github.com/bincrafters/conan-yasm_installer"
+    description = "Yasm is a complete rewrite of the NASM assembler under the “new” BSD License"
+    license = "BSD"
+    exports_sources = ["LICENSE"]
+    settings = "os_build", "arch_build", "compiler"
+    _source_subfolder = "sources"
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = 'yasm-%s' % self.version
+        os.rename(extracted_dir, self._source_subfolder)
+        tools.download('https://raw.githubusercontent.com/yasm/yasm/master/YASM-VERSION-GEN.bat',
```

Comment:
Why is here an extra download happening? shouldn't be all relevant files be already in the download archive?

Even if not, downloading from master seems definitely like a bad idea

Reply:
the file is missing from the released archive. I may change it to use specific revision/tag (or alternatively, just put file near to the recipe?)

Reply:
Yes, please, use a the commit sha1. You can use this one, which is the latest:

https://github.com/yasm/yasm/blob/bcc01c59d8196f857989e6ae718458c296ca20e3/YASM-VERSION-GEN.bat